avatars/models.py

```diff
@@ -170,19 +180,34 @@ class AvatarizationResultResponse(BaseModel):
     privacy_metrics: PrivacyMetrics
     signal_metrics: SignalMetrics
     avatars_dataset: DatasetResponse
-    sensitive_unshuffled_avatars_datasets: DatasetResponse
+    sensitive_unshuffled_avatars_dataset: DatasetResponse
 
 
 class Job(BaseModel):
     id: UUID = Field(..., title="Id")
     status: JobStatus
-    dataset_id: UUID = Field(..., title="Dataset Id")
+    dataset_id: Optional[UUID] = Field(
+        None,
+        description="Deprecated, use parameters.dataset_id instead.",
+        title="Dataset Id",
+    )
     error_message: Optional[str] = Field(None, title="Error Message")
     traceback: Optional[str] = Field(None, title="Traceback")
     result: Optional[AvatarizationResultResponse] = None
+    parameters: AvatarizationParameters
     current_progress: Optional[JobProgress] = None
 
 
 class JobCreate(BaseModel):
-    dataset_id: UUID = Field(..., title="Dataset Id")
+    kind: Optional[JobKind] = "avatarization"
+    dataset_id: UUID = Field(
+        ...,
+        description="Deprecated. Use AvatarizationJobCreate with dataset_id in AvatarizationParameters.",
```

Comment:
The whole class is deprecated, not the `dataset_id` field !

Reply:
Yes, but I can't find a way to keep the class docstring from the API to appear here.

Reply:
Done.